website/docs/docs/core/source-install.md

```diff
@@ -12,28 +12,58 @@ Downloading and building dbt Core will enable you to contribute to the project b
 
 ### Installing dbt Core
 
-To install `dbt-core` from the GitHub code source:
+Beginning in v1.8, you must install both dbt Core and the desired adapter because `dbt-core` has been decoupled from the adapter installation.
```

Comment:
```suggestion
Beginning in v1.8, installing an adapter no longer automatically installs `dbt-core`. This is because adapters and core versions have been decoupled from each other so we no longer want to overwrite your existing dbt-core installation.
```
Not sure but this might be a bit more precise about what's happening

Reply:
Thank you, Colin!

Reply:
Addressed in https://github.com/dbt-labs/docs.getdbt.com/pull/5002/commits/ac9737b67592a9669e4e365851d3284d2237d1ed